app.rambox.Rambox.yaml

```
@@ -0,0 +1,63 @@
+app-id: app.rambox.Rambox
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+rename-appdata-file: rambox.metainfo.xml
+rename-desktop-file: rambox.desktop
+rename-icon: rambox
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --share=ipc
+  ## necessary for webcam
+  - --device=all
+  - --socket=pulseaudio
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.freedesktop.portal.Background
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --filesystem=xdg-config/kdeglobals:ro
+  - --filesystem=xdg-config/autostart:create
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-download
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-videos
+  - --filesystem=xdg-run/pipewire-0
+command: rambox
+separate-locales: false
+modules:
+  - name: rambox
+    buildsystem: simple
+    sources:
+      - type: file
+        dest-filename: rambox.deb
+        url: https://github.com/ramboxapp/community-edition/releases/download/0.7.9/Rambox-0.7.9-linux-amd64.deb
+        sha256: 04b9aaae7ced1e1ba71800bea861a45e57fed73a53923d263e7cb173a04c779f
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/ramboxapp/community-edition/releases/latest
+          version-query: .tag_name
+          url-query: .assets[] | select(.name=="Rambox-"+ $version +"-linux-amd64.deb")
+            | .browser_download_url
+      - type: script
+        dest-filename: rambox.sh
+        commands:
+          - export TMPDIR="$XDG_RUNTIME_DIR/app/${FLATPAK_ID}"
+          - zypak-wrapper /app/share/rambox/rambox $@
+      - type: file
+        path: rambox.metainfo.xml
+    build-commands:
+      - install -Dm755 rambox.sh /app/bin/rambox
+      - install -Dm644 rambox.metainfo.xml -t /app/share/metainfo
```

```suggestion
      - install -Dm644 app.rambox.Rambox.metainfo.xml -t /app/share/metainfo
```

As I said previously.

---

already fixed, but not pushed yet